utils/build/docker/python/install_ddtrace.sh

```diff
@@ -7,8 +7,8 @@ cd /binaries
 if [ -e "dd-trace-py" ]; then
     echo "Install from local folder /binaries/dd-trace-py"
     pip install /binaries/dd-trace-py
-elif [ "$(ls *.whl *.tar.gz | wc -l)" = "1" ]; then
-    path=$(readlink -f $(ls *.whl *.tar.gz))
+elif [ "$(ls *.whl | wc -l)" = "1" ]; then
```

Comment:
why removing `.tar.gz` ?

Reply:
fixed this

Reply:
actually, the tests were failing on docker pull for PHP and .NET. The problem is because we are building the tracer buddies, however, these repos have `.tar` files that end up causing the check on line 10 to pass, which is incorrect, and tries to build using the tar file. I've added a whitelist of these file names but maybe this should be fixed some other way.